tools/ci_build/github/windows/post_code_coverage_to_dashboard.py

```diff
@@ -10,27 +10,30 @@
 
 import argparse
```

Comment:
looks like this is called from Linux now, maybe tools/ci_build/github/windows is not the best location. can address it in a separate PR